tests/serialization/test_parameter_serialization.py

```diff
@@ -7,27 +7,30 @@
 
 from baybe.parameters.base import Parameter
 
+from ..conftest import _CHEM_INSTALLED
 from ..hypothesis_strategies.parameters import (
     categorical_parameter,
     custom_parameter,
     numerical_continuous_parameter,
     numerical_discrete_parameter,
-    substance_parameter,
     task_parameter,
 )
 
+parameter_strategies = [
+    param(numerical_discrete_parameter(), id="NumericalDiscreteParameter"),
+    param(numerical_continuous_parameter(), id="NumericalContinuousParameter"),
+    param(categorical_parameter(), id="CategoricalParameter"),
+    param(task_parameter(), id="TaskParameter"),
+    param(custom_parameter(), id="CustomParameter"),
+]
```

Comment:
@AdrianSosic a first try using your suggested way via skipif (which I would also prefer) didnt work. It seems during the collection the object is still instantiated, resulting in imprt error even if its marked as skip